packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -405,6 +407,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
     _selectionPainter.dispose();
     _caretPainter.dispose();
     _textPainter.dispose();
+    _internalShowCursor?.dispose();
```

Comment:
There is `set showCursor(ValueNotifier<bool> value) {`. It makes things little more complicated. Should the setter dispose previous value?
Instead of creating new member `_internalShowCursor`, should we create flag `_disposeShowCursor` and update it every time _showCursor is assigned?

Reply:
@polina-c, thanks for the review. 
I see what you mean, but we need to care only about the first internal `ValueNotifier`, which gets created on `RenderEditable` creation if  `showCursor` is not passed. All next `showCursor` `ValueNotifier`s will be set using a setter, and their disposal is not our responsibility. 
Am I missing something?)

Reply:
True, however, if setter is invoked, should we dispose the previous value right away and null out all references to it, without waiting for disposal of the entire class?

Reply:
@polina-c great thought! 
Will do it that way, thanks :)